### DISCUSSION TITLE
v2.0.0-beta04について気づいたことがあれば

### DISCUSSION THREAD

**Discussion by keiji** [cocoa-mhlw/cocoa#903] [General]
v2.0.0-beta04（クローズド）に関するDiscussionです。
iOS版、Android版ともに配信開始済。

 * v2.0.0のbeta版で通知を受け取っても行政検査の対象にならないことを明記しました（v2.0.0より後のバージョンのbetaでは対象にしたい）
 * その他、画面デザインを調整しました。

**Comment by polygon0323** [comment]
解像度の低い端末で、説明文が欠けていました。

Jelly 2 (480x854)
Android 10
フォントサイズ設定はデフォルトです。

以下は一例です。
![Screenshot_20220316-121052](https://user-images.githubusercontent.com/4208047/158523428-571b7895-3898-4c0b-97dd-0097deabd0a7.png)

何かこちらで試せることがありましたらお知らせください。

**Reply by keiji** [reply]
ありがとうございます！

お使いのバージョンはオープンベータでしょうか？　ただこの画面はv2.0.0でも手を付けていないので、v1.4.1でも発生しますね。
確認してIssue作ります！

**Reply by polygon0323** [reply]
オープンベータの方です。トピック誤りですみません。
移動させた方がよければお知らせください。
また、「接触の確認方法」画面も欠けがあるようです。

**Reply by keiji** [reply]
こちらこそオープンベータのDiscussionを作っておらず失礼しました。

本件、Issueとして対応します（ #923 ）！

**Reply by keiji** [reply]
@polygon0323 
こちら検討を進めています。
もしご存じであれば、Jelly 2のピクセル密度を教えて頂けませんか？（`xxx-hdpi`とか`mdpi`とかそんなやつです）。

**Reply by polygon0323** [reply]
DisplayInfoアプリで見たところ、 `tvdpi` と言われました。
![image](https://user-images.githubusercontent.com/4208047/159261664-c5b88552-5d1a-4083-892c-203f8bd3731f.png)

**Reply by keiji** [reply]
ありがとうございます！　`tvdpi`、スマートフォンで使われているのは初めて見ました……！

**Comment by kvaluation** [comment]
@keiji  複数の系列の管理をありがとうございます。

■「午前」の経緯
https://github.com/cocoa-mhlw/cocoa/discussions/865#discussioncomment-2200478

ご関係者各位
午前9時00分と、「午前」が明記されたバージョン（v2.0.0-beta04）のスクリーンショット、共有させてください。

■「受信した信号」表示例のスクショ（オープンβの画面とは異なります）
<img src="https://user-images.githubusercontent.com/60703087/159441275-d2b71be4-d3de-404c-84cd-ed11435ff023.png" width="400px)" style="max-width: 100%;"></a>

■ 広範な接触（低リスクすれ違い）スクショ（オープンβの画面とは異なります）
3月13日のすれ違いが表示されました。
MatchCountやrollingStartNumberから分かる情報（広範な接触があった事実と、その接触日UTC）を自然に知ることができて、とても良いと思います。
<img src="https://user-images.githubusercontent.com/60703087/159441953-3fc6bcc7-42be-4236-9352-b8f7ee6eb9a0.png" width="400px)" style="max-width: 100%;"></a>
